src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -59,6 +59,7 @@
     "Token": ""
   },
   "AppInsights": {
+    "EncryptionKey": ""
```

Comment:
Is this EncryptionKey set by us to decrypt the apikey? How do we know which key to use?

Reply:
Yes, we will set in manually here and in backend project of portal code